scan/lib/scan/test_command_generator.rb

```diff
@@ -37,6 +37,7 @@ def options
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
         options << "-enableCodeCoverage YES" if config[:code_coverage]
         options << "-enableAddressSanitizer YES" if config[:address_sanitizer]
```

Comment:
Done. Removed default `false` value for all 3 options.
If users don't specify this option, we simply add no `-enable***` flag to `xcodebuild` and it uses whatever is already set in Xcode project or scheme.